landing_page/templates/landing_page/about.html

```diff
@@ -19,6 +20,33 @@ <h2>What is ReeldIn?</h2>
       provide the best recommendations possible. We're here to make your
       movie-watching experience as enjoyable as possible.
     </p>
+
+    <br />
+    <h2>FAQ</h2>
+    <br />
+    <h3>Q: How do I make an account?<br /></h3>
+    <br />
+    <p>
+      A: To make an account, click the "Login" button in the top right corner of
+      the page. You'll be prompted to input your email and password. Once you've
+      done that, you'll have access to all of ReeldIn's features.
+    </p>
+    <br />
+    <h3>Q: How do I get recommendations? <br /></h3>
+    <br />
+    <p>
+      A: To get recommendations, click the "Get Recommendations" button on the
+      home page. You'll be prompted to input your favorite movies, and ReeldIn
```

Comment:
need to revisit this as the 'get recommendations' button, by default, leads to twenty questions.

i think it would be worth exploring the idea of 'do you want to query our database' or 'do you want to watch something based on what you've seen', buttons for that (like one on either side of that home page) or a different page that then redirects you from there

Reply:
Totally, we can change things around. I think we can have another page for getting recommendations and/or include those buttons in the user's profile so that they can quickly transition from viewing movies they've added to their lists to checking out more movies.